lambda_uploader/package.py

```diff
@@ -49,6 +49,11 @@ def build_package(path, requires, virtualenv=None, ignore=[],
     return pkg
 
 
+def create_package(path, zipfile_name=ZIPFILE_NAME):
```

Comment:
Can we add a docstring to this just to clarify what it does. I don't want people that are using the lambda-uploader as an API to be confused.